training/train.py

```diff
@@ -302,30 +302,48 @@ def create_git_pr(s3_client, model_version, recall_at_30):  # pragma: no cover
     prev_hyperparams = s3_client.read_json_file(k)
 
     # Convert the json description to string
-    description = json.dumps(prev_hyperparams).replace('"', '\\"')
+    description = 'Previous hyper parameters :: ' + json.dumps(prev_hyperparams).replace('"', '\\"')
 
     prev_recall = prev_hyperparams.get('recall_at_30', 0.55)
+    _logger.info('create_git_pr:: Prev => Model {}, Recall {}  Curr => Model {}, Recall {}'.format(
+        previous_version, prev_recall, model_version, recall_at_30))
     if recall_at_30 >= prev_recall:
         try:
             # Invoke bash script to create a saas-analytics PR
-            t = subprocess.Popen(['sh', 'rudra/utils/github_helper.sh', 'f8a-hpf-insights.yaml',
-                                  'MODEL_VERSION', str(model_version), description],
-                                 shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            # Wait for the subprocess to get over
-            t.wait(60)
-            if t.returncode == 0:
-                _logger.info("Successfully created a PR")
+            t1 = subprocess.Popen(['wget', '-v',
+                                   'https://raw.githubusercontent.com/fabric8-analytics/'
```

Comment:
I might be wrong here, but it is bit weird to do all these things to raise a PR. Unpythonic I would say. I would recommend to use Github sdk or gh cli(still unconventional way).

Reply:
@arajkumar I agree to your comment, but as I updated in other PR, refactoring code is next step, I have raised a separate ticket for that. Primary goal was to fix the issue in the current re-training flow to make it up and running as it was a year back.